File: Back/src/main/java/com/showbook/back/entity/MemberImage.java
package com.showbook.back.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class MemberImage {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "member_image_id")
	private Long id;

	@Column(nullable = false)
	private String memberImageName;

	@Column(nullable = false)
	private String originalImageName;

	@Column(name="member_image_url", nullable = false)
	private String imageUrl;

	@Builder
	public MemberImage(Long id, String memberImageName, String originalImageName, String imageUrl) {
		this.id = id;
		this.memberImageName = memberImageName;
		this.originalImageName = originalImageName;
		this.imageUrl = imageUrl;
	}
}

File: ml/flask/models/MemberImage.py
from config.dbConfig import db

class MemberImage(db.Model):
    '''
    member_image_id : pk
    member_image_url : 회원 사진이 저장된 url
    member_image_name : 회원 사진 이름
    original_image_name : 원래 회원 사진 이름
    '''
    __tablename__ = "member_image"
    
    member_image_id = db.Column(db.Integer,nullable = False,autoincrement=True, primary_key=True)
    member_image_url = db.Column(db.String(255),nullable = False)
    genmember_image_nameder = db.Column(db.String(255),nullable = False)
    original_image_name = db.Column(db.String(255),nullable = False)